test/providers.test.js

```diff
@@ -328,4 +328,11 @@ test('JsonRpc connection object exist without url provided', async () => {
 test('JsonRpc connection url is not null on empty string', async () => {
     const provider = new nearApi.providers.JsonRpcProvider('');
     expect(provider.connection).toStrictEqual({ url: '' });
+});
+
+test('near json rpc fetch node status', async () => {
+    const config = Object.assign(require('./config')(process.env.NODE_ENV || 'test'));
```

Comment:
What does the `Object.assign` do here? From what I've seen from the docs, it should take a target and some sources, but it looks like we're only supplying the target. Wouldn't this:
```suggestion
    const config = require('./config')(process.env.NODE_ENV || 'test');
```
be the same? Or is there some magic going on I'm not seeing here

Reply:
Yes, it will work the same way, thanks!